igor-web/src/test/groovy/com/netflix/spinnaker/igor/concourse/service/ConcourseServiceSpec.groovy

```diff
@@ -0,0 +1,334 @@
+{"data":{"origin":{"id":"5ed14567"},"time":1591654846,"exit_status":0,"version":{"digest":"sha256:30bf1745fc5884469fb61f1652374e2f1befc9e937754a0b9914fe886111c96d"},"metadata":[{"name":"image","value":"sha256:81dcf"}]},"event":"finish-put","version":"5.1"},
+{"data":{"origin":{"id":"5ed14568"},"time":1591654866,"exit_status":0,"version":{"digest":"sha256:30bf1745fc5884469fb61f1652374e2f1befc9e937754a0b9914fe886111c96d"},"metadata":[{"name":"image","value":"sha256:81dcf"}]},"event":"finish-get","version":"5.1"},
+{"data":{"origin":{"id":"5ed1456a"},"time":1591654894,"exit_status":0,"version":{"path":"artifacts/gateway-service/deploy-1.0.178-65a7c12.yaml"},"metadata":[{"name":"filename","value":"deploy-1.0.178-65a7c12.yaml"},{"name":"url","value":"https://s3-us-west-2.amazonaws.com/artifacts-bucket/artifacts/gateway-service/deploy-1.0.178-65a7c12.yaml"}]},"event":"finish-put","version":"5.1"},
+{"data":{"origin":{"id":"5ed1456b"},"time":1591654896,"exit_status":0,"version":{"path":"artifacts/gateway-service/deploy-1.0.178-65a7c12.yaml"},"metadata":[{"name":"filename","value":"deploy-1.0.178-65a7c12.yaml"},{"name":"url","value":"https://s3-us-west-2.amazonaws.com/artifacts-bucket/artifacts/gateway-service/deploy-1.0.178-65a7c12.yaml"}]},"event":"finish-get","version":"5.1"},
+{"data":{"origin":{"id":"5ed1456d"},"time":1591654924,"exit_status":0,"version":{"number":"1.0.178"},"metadata":[{"name":"number","value":"1.0.178"}]},"event":"finish-put","version":"5.1"},
+{"data":{"origin":{"id":"5ed1456e"},"time":1591654926,"exit_status":0,"version":{"number":"1.0.178"},"metadata":[{"name":"number","value":"1.0.178"}]},"event":"finish-get","version":"5.1"},
+{"data":{"origin":{"id":"5ed14570"},"time":1591654959,"exit_status":0,"version":{"ref":"65a7c129fe402634ee1d94b5ba2577fd46a039f2"},"metadata":[{"name":"commit","value":"65a7c129fe402634ee1d94b5ba2577fd46a039f2"},{"name":"author","value":"John Smith"},{"name":"author_date","value":"2020-06-08 15:13:23 -0700"},{"name":"branch","value":"master"},{"name":"tags","value":"v1.0.178"},{"name":"message","value":"DOOL-1990: reopening question sets completionPercent to zero\\n\\n"},{"name":"url","value":"https://github.com/myteam/gateway/commit/65a7c129fe402634ee1d94b5ba2577fd46a039f2"}]},"event":"finish-put","version":"5.1"},
+{"data":{"origin":{"id":"5ed14571"},"time":1591654963,"exit_status":0,"version":{"ref":"65a7c129fe402634ee1d94b5ba2577fd46a039f2"},"metadata":[{"name":"commit","value":"65a7c129fe402634ee1d94b5ba2577fd46a039f2"},{"name":"author","value":"John Smith"},{"name":"author_date","value":"2020-06-08 15:13:23 -0700"},{"name":"branch","value":"master"},{"name":"tags","value":"v1.0.178"},{"name":"message","value":"DOOL-1990: reopening question sets completionPercent to zero\\n\\n"},{"name":"url","value":"https://github.com/myteam/gateway/commit/65a7c129fe402634ee1d94b5ba2577fd46a039f2"}]},"event":"finish-get","version":"5.1"}
+        ]'''
```

Comment:
Nit: Consider indenting the lines and rather add a `.stripIndent()` after the ending triple quotes. Same with the json in the `planFixture()` method as well.

Reply:
done